ArduCopter/Copter.h

```diff
@@ -686,7 +686,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     float get_look_ahead_yaw();
     void update_throttle_hover();
     void set_throttle_takeoff();
-    float get_pilot_desired_throttle(int16_t throttle_control);
+    float get_pilot_desired_throttle(int16_t throttle_control, float thr_mid);
```

Comment:
@OXINARF, ok, I was thinking about that.. but we don't use defaults much.. anyway, i'll make that change, thanks!